api/proto/info/info.proto

```diff
@@ -504,9 +506,44 @@ message ZInfoNetworkInstance {
     ZInfoVpn vinfo = 31;
     ZInfoLisp linfo = 32;
   }
+  UsageInfo usage = 33;
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
+  google.protobuf.Timestamp lastUseTime = 3; // When refCount dropped to zero
```

Comment:
lastUseTime = 3; // When refCount dropped to zero

lastUseTime --> This name is a bit confusing... This sounded more like the file last use time.. Which is not the intention here.. This is the time when the last of the objects stopped using this..

And if the refcount increases, if this supposed to be set to 0?

Reply:
@kalyan-nidumolu lastRefcountChangeTime sounds like a better name

Reply:
I will change it to lastRefCountChangeTime per @eriknordmark suggestion.

Reply:
Actually @kalyan-nidumolu suggested that (and I promptly updated the API document)

Reply:
Ack, I changed the field name per @kalyan-nidumolu suggestion.